wrapper/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/rs/cors/wrapper/gin
```

Comment:
@komuw: shouldn't this file be at `wrapper/gin` level?

Reply:
@rs agreed. 
I have done that now.